tests/StacktraceTest.php

```diff
@@ -457,6 +457,19 @@ public function testFromBacktraceWithAnonymousClass(): void
         );
     }
 
+    public function testFromBacktraceWithAnonymousClassTrimsPath(): void
```

Comment:
I think it would be more appropriate to merge this test with the one right above so that we test in one place all possible outcomes of a frame involving an anonymous class. Also unrelated, but would you be so kind to rename all test methods that starts with `testFromBacktrace` to `testCreateFromBacktrace` 🙏?